cmd/bosun/expr/funcs.go

```diff
@@ -68,6 +85,15 @@ var Graphite = map[string]parse.Func{
 	},
 }
 
+var LogstashElastic = map[string]parse.Func{
```

Comment:
Let's put all the elastic stuff in its own file. I'm going to move the tsdb and graphite stuff around, also.